services/frontend-v3/src/components/profileForms/primaryInfoForm/PrimaryInfoFormView.jsx

```diff
@@ -425,6 +470,140 @@ const PrimaryInfoFormView = ({
     return undefined;
   };
 
+  const handleGedsConfirm = async () => {
+    await axios
+      .put(
```

Comment:
Please use the Axios instance created in the frontend. It is called axios-instance.js just import that instead of axioms. You will only need to add the URL `api/....` no need to add the backend URL. You can see examples in the other components.

Reply:
Now use axios-instance